tasks/tasks.go

```diff
@@ -38,12 +40,37 @@ func LoadStorageEngine(ctx context.Context, config *tsbridge.Config) (storage.Ma
 	}
 }
 
-// Sync updates all configured metrics.
-func Sync(ctx context.Context, config *tsbridge.Config, metrics *tsbridge.Metrics, store storage.Manager) error {
-	metricCfg, err := tsbridge.NewMetricConfig(ctx, config, store)
+// Checks if the metric config file has been updated, and updates the cached file stats if it has.
+func metricConfigFileUpdated(ctx context.Context, config *tsbridge.Config, metricCfgFs *os.FileInfo) (bool, error) {
+	currMetricCfgFs, err := os.Stat(config.Options.Filename)
+	if err != nil {
+		return false, err
+	}
+	if (*metricCfgFs).ModTime() != currMetricCfgFs.ModTime() && (*metricCfgFs).Size() != currMetricCfgFs.Size() {
```

Comment:
Is there an instance where the `Size()` would change, and the `ModTime()` wouldn't?

you could just do

`return (*metricCfgFs).ModTime() != currMetricCfgFs.ModTime()), nil`